src/main/java/appeng/container/slot/OptionalSlotFake.java

```diff
@@ -26,12 +26,12 @@
 public class OptionalSlotFake extends SlotFake
 {
 
-	public final int srcX;
-	public final int srcY;
-	final int invSlot;
-	final int groupNum;
-	final IOptionalSlotHost host;
-	public boolean renderDisabled = true;
+	private final int srcX;
+	private final int srcY;
+	private final int invSlot;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "invSlot" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)